servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/HttpObjectDecoder.java

```diff
@@ -225,6 +226,9 @@ protected final void decode(final ChannelHandlerContext ctx, final ByteBuf buffe
                 currentState = State.READ_INITIAL;
             }
             case READ_INITIAL: {
+                if (!buffer.isReadable()) {
```

Comment:
IIUC this is just an optimization to avoid attempting to search for CRLF? is this thought to be common?

Reply:
Can not say it's common, because usually TCP packet includes enough data for the start-line. This is to prevent going inside `handlePartialInitialLine` when CRLF was not found. Let me move it inside the `if` statement.

Reply:
Actually, I don't think we need it. Let me kill it :)